### ISSUE TITLE
ERROR: Task generation 'render_site' has duplicated definition of 'render_listings'

### ISSUE THREAD

**Issue by rogerbinns** [getnikola/nikola#682]
I updated my conf.py to address all the deprecation issues.  Afterwards I get this.  I also get it before making any changes with a conf.py that worked with v5 but then get 8 unrelated warnings.

```
$ nikola build
Scanning posts....done!
ERROR: Task generation 'render_site' has duplicated definition of 'render_listings'
```

**Comment by smurfix** [comment]
Meh. I have something like this error too, and I installed current nikola from a (clean) git copy.

    ERROR: Task generation 'render_site' has duplicated definition of 'render_indexes:output/en/index.html'

I then commented off STORY_INDEX=True in conf.py. The build worked.

**Comment by jwhendy** [comment]
I'm happy to open a new issue if you'd like, but just ran into this myself. I added functionality to parse `.org` files with the `orgmode` plugin. I figured out that the issue was `index.rst` (my original test as I'm just getting started with `nikola`) conflicting with `index.org`, even though I changed the slug definition.

My intuition based on various notes in the conf.py is that `..slug` changes the output location, but having both `index.rst` and `index.org` definitely was the issue. I changed the former's name and all was well.

Probably as a practice, the same names with different extensions isn't a good practice, but it *seemed* like a potential bug if they had different `..slug` definitions.
